GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/MountTests.cs

```diff
@@ -83,6 +85,8 @@ public void MountSetsCoreHooksPath()
                 this.Enlistment.RepoRoot, "config core.hookspath")
                 .Trim('\n')
                 .ShouldEqual(expectedHooksPath);
+
+            GVFSHelpers.HandleUnregisterForOfflineIO();

```

Comment:
Should `GVFSHelpers.HandleUnregisterForOfflineIO()` be put in the `finally` of a `try/finally` so that if there is an exception thrown in the test code we still call `HandleUnregisterForOfflineIO`?